tools/dcounter.c

```diff
@@ -22,7 +22,7 @@ int main(int argc, char **argv) {
     }
     if (argc == 2 && strstr(*argv, "-l")) {
         newline = 1;
-        strncpy (prefix,*(argv+1),strlen(*(argv+1)));
+        strncpy (prefix, *(argv+1), strlen(prefix));
```

Comment:
I haven't deeply looked the code here, but I guess this change is fine because source and destination strings are expected the have the same size. In any case now looks good.